frontend/src/components/AllocationInfo/AllocationInfo.js

```diff
@@ -61,6 +61,7 @@ class AllocationInfo extends Component {
                     <TableRowColumn style={{ width: 180 }}>{element.Type}</TableRowColumn>
                     <TableRowColumn>
                       {element.Message ||
+                        element.DisplayMessage ||
```

Comment:
maybe we should add it up as the first item - so change the order of Message and DisplayMessage

Reply:
@jippi thanks, I have applied the changes, let me know if something else needs to be done for this merge request.